server/views/referrals/riskInformation.njk

```diff
@@ -33,6 +33,27 @@
 
         {{ govukDetails(riskLevelDetailsArgs) }}
 
+        {% if presenter.text.whoIsAtRisk or presenter.text.natureOfRisk or presenter.text.riskImminence %}
```

Comment:
No biggie, but I'd probably encapsulate this in a method like `shouldShowRiskWarning` on the presenter.